lib/streamlit/type_util.py

```diff
@@ -218,6 +218,14 @@ def get_fqn_type(obj: object) -> str:
 _SNOWPARK_DF_ROW_TYPE_STR: Final = "snowflake.snowpark.row.Row"
 _SNOWPARK_TABLE_TYPE_STR: Final = "snowflake.snowpark.table.Table"
 _PYSPARK_DF_TYPE_STR: Final = "pyspark.sql.dataframe.DataFrame"
+_MODIN_DF_TYPE_STR: Final = "modin.pandas.dataframe.DataFrame"
+_MODIN_SERIES_TYPE_STR: Final = "modin.pandas.Series.Series"
+_SNOWPANDAS_DF_TYPE_STR: Final = (
+    "snowflake.snowpark.modin.pandas.frontend.dataframe.DataFrame"
+)
+_SNOWPANDAS_SERIES_TYPE_STR: Final = (
+    "snowflake.snowpark.modin.pandas.frontend.series.Series"
+)
```

Comment:
```suggestion
_SNOWPANDAS_DF_TYPE_STR: Final = (
    "snowflake.snowpark.modin.pandas.dataframe.DataFrame"
)
_SNOWPANDAS_SERIES_TYPE_STR: Final = (
    "snowflake.snowpark.modin.pandas.series.Series"
)
```
I'm not sure what version of Snowpark pandas you're testing against, but these are the paths in our most recent version.

Reply:
@sfc-gh-joshi Thanks for the comment 👍 I believe I tried it with `1.14.0a1`. What is the latest version?

Reply:
We just released `1.14.0a2`, which should have this change.

Reply:
sounds good, I updated the the paths.